packages/elements/src/datetime-picker/constants.ts

```diff
@@ -0,0 +1,6 @@
+enum DateTimePickerDataName {
+  from = 'from',
+  to = 'to'
```

Comment:
Let's use `FROM` & `TO` so there is no need to convert their case later.

Reply:
Fixed